We have occasional timeouts for migration on ARM sqlite tests (compat)

Usually compat tests:

```
_______________________________________________________________ ERROR at setup of TestRemoteCeleryControlCommands.test_list_celery_workers _______________________________________________________________
/usr/local/lib/python3.10/site-packages/pytest_asyncio/plugin.py:681: in pytest_fixture_setup
    return (yield)
/usr/local/lib/python3.10/site-packages/tests_common/pytest_plugin.py:411: in initialize_airflow_tests
    _initialize_airflow_db(request.config.option.db_init, airflow_home)
/usr/local/lib/python3.10/site-packages/tests_common/pytest_plugin.py:436: in _initialize_airflow_db
    initial_db_init()
/usr/local/lib/python3.10/site-packages/tests_common/test_utils/db.py:118: in initial_db_init
    get_auth_manager().init()
/usr/local/lib/python3.10/site-packages/airflow/providers/fab/auth_manager/fab_auth_manager.py:204: in init
    self._sync_appbuilder_roles()
/usr/local/lib/python3.10/site-packages/airflow/providers/fab/auth_manager/fab_auth_manager.py:571: in _sync_appbuilder_roles
    self.security_manager.sync_roles()
/usr/local/lib/python3.10/site-packages/airflow/providers/fab/auth_manager/security_manager/override.py:1332: in sync_roles
    self.bulk_sync_roles(self.ROLE_CONFIGS)
/usr/local/lib/python3.10/site-packages/airflow/providers/fab/auth_manager/security_manager/override.py:1466: in bulk_sync_roles
    self.add_permission_to_role(role, perm)
/usr/local/lib/python3.10/site-packages/airflow/providers/fab/auth_manager/security_manager/override.py:1962: in add_permission_to_role
    self.get_session.commit()
/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/session.py:1454: in commit
    self._transaction.commit(_to_root=self.future)
/usr/local/lib/python3.10/site-packages/sqlalchemy/orm/session.py:839: in commit
    trans.commit()
/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py:2469: in commit
    self._do_commit()
/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py:2659: in _do_commit
    self._connection_commit_impl()
/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py:2630: in _connection_commit_impl
    self.connection._commit_impl()
/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py:1096: in _commit_impl
    self._handle_dbapi_exception(e, None, None, None, None)
/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py:2138: in _handle_dbapi_exception
    util.raise_(exc_info[1], with_traceback=exc_info[2])
/usr/local/lib/python3.10/site-packages/sqlalchemy/util/compat.py:211: in raise_
    raise exception
/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/base.py:1094: in _commit_impl
    self.engine.dialect.do_commit(self.connection)
/usr/local/lib/python3.10/site-packages/sqlalchemy/engine/default.py:686: in do_commit
    dbapi_connection.commit()
E   Failed: Timeout >60.0s
```

Example https://github.com/apache/airflow/actions/runs/16827137062/job/47666872315

--- a/providers/celery/tests/unit/celery/cli/test_celery_command.py
+++ b/providers/celery/tests/unit/celery/cli/test_celery_command.py
@@ -534,6 +534,15 @@ class TestFlowerCommand:
 
 
 class TestRemoteCeleryControlCommands:
+    @pytest.fixture(autouse=True)
+    def _disable_cli_action_logging(self):
+        # Keep these tests as true unit tests: action_cli's default callbacks write to DB.
+        with (
+            patch("airflow.utils.cli.cli_action_loggers.on_pre_execution"),
+            patch("airflow.utils.cli.cli_action_loggers.on_post_execution"),
+        ):
+            yield
+
     @classmethod
     def setup_class(cls):
         with conf_vars({("core", "executor"): "CeleryExecutor"}):
@@ -541,7 +550,6 @@ class TestRemoteCeleryControlCommands:
             importlib.reload(cli_parser)
             cls.parser = cli_parser.get_parser()
 
-    @pytest.mark.db_test
     @mock.patch("airflow.providers.celery.executors.celery_executor.app.control.inspect")
     def test_list_celery_workers(self, mock_inspect):
         args = self.parser.parse_args(["celery", "list-workers", "--output", "json"])
@@ -559,7 +567,6 @@ class TestRemoteCeleryControlCommands:
             assert key in celery_workers[0]
         assert any("celery@host_1" in h["worker_name"] for h in celery_workers)
 
-    @pytest.mark.db_test
     @mock.patch("airflow.providers.celery.executors.celery_executor.app.control.shutdown")
     def test_shutdown_worker(self, mock_shutdown):
         args = self.parser.parse_args(["celery", "shutdown-worker", "-H", "celery@host_1"])
@@ -569,7 +576,6 @@ class TestRemoteCeleryControlCommands:
             celery_command.shutdown_worker(args)
             mock_shutdown.assert_called_once_with(destination=["celery@host_1"])
 
-    @pytest.mark.db_test
     @mock.patch("airflow.providers.celery.executors.celery_executor.app.control.broadcast")
     def test_shutdown_all_workers(self, mock_broadcast):
         args = self.parser.parse_args(["celery", "shutdown-all-workers", "-y"])
@@ -579,7 +585,6 @@ class TestRemoteCeleryControlCommands:
             celery_command.shutdown_all_workers(args)
             mock_broadcast.assert_called_once_with("shutdown")
 
-    @pytest.mark.db_test
     @mock.patch("airflow.providers.celery.executors.celery_executor.app.control.add_consumer")
     def test_add_queue(self, mock_add_consumer):
         args = self.parser.parse_args(["celery", "add-queue", "-q", "test1", "-H", "celery@host_1"])
@@ -589,7 +594,6 @@ class TestRemoteCeleryControlCommands:
             celery_command.add_queue(args)
             mock_add_consumer.assert_called_once_with("test1", destination=["celery@host_1"])
 
-    @pytest.mark.db_test
     @mock.patch("airflow.providers.celery.executors.celery_executor.app.control.cancel_consumer")
     def test_remove_queue(self, mock_cancel_consumer):
         args = self.parser.parse_args(["celery", "remove-queue", "-q", "test1", "-H", "celery@host_1"])
@@ -599,7 +603,6 @@ class TestRemoteCeleryControlCommands:
             celery_command.remove_queue(args)
             mock_cancel_consumer.assert_called_once_with("test1", destination=["celery@host_1"])
 
-    @pytest.mark.db_test
     @mock.patch("airflow.providers.celery.executors.celery_executor.app.control.cancel_consumer")
     @mock.patch("airflow.providers.celery.executors.celery_executor.app.control.inspect")
     def test_remove_all_queues(self, mock_inspect, mock_cancel_consumer):

